openml/extensions/sklearn/extension.py

```diff
@@ -1133,17 +1200,29 @@ def _prediction_to_probabilities(
 
         try:
             # for measuring runtime. Only available since Python 3.3
-            if can_measure_runtime:
-                modelfit_starttime = time.process_time()
+            modelfit_start_cputime = None
+            modelfit_duration_cputime = None
+            modelpredict_start_cputime = None
+
+            modelfit_start_walltime = None
+            modelfit_duration_walltime = None
+            modelpredict_start_walltime = None
+            if can_measure_cputime:
+                modelfit_start_cputime = time.process_time()
+            if can_measure_wallclocktime:
+                modelfit_start_walltime = time.time()
 
             if isinstance(task, OpenMLSupervisedTask):
                 model_copy.fit(train_x, train_y)
             elif isinstance(task, OpenMLClusteringTask):
                 model_copy.fit(train_x)
 
-            if can_measure_runtime:
-                modelfit_duration = (time.process_time() - modelfit_starttime) * 1000
-                user_defined_measures['usercpu_time_millis_training'] = modelfit_duration
+            if can_measure_cputime:
+                modelfit_duration_cputime = (time.process_time() - modelfit_start_cputime) * 1000
+                user_defined_measures['usercpu_time_millis_training'] = modelfit_duration_cputime
+            if can_measure_wallclocktime:
+                modelfit_duration_walltime = (time.time() - modelfit_start_walltime) * 1000
```

Comment:
[`time.time`](https://docs.python.org/3/library/time.html#time.time) is in seconds, I think the `* 1000` is a mistake.

Reply:
we record millis :)